### ISSUE TITLE
Update FTE Vignette to align with new FTE sql 

### ISSUE THREAD

**Issue by andreabringhurst** [ut-effectiveness/utDataStoR#48]
Pull request #22 makes changes to the FTE queries. The vignette needs to be updated to align with these queries. The FTE vignette needs to look similar to the headcount vignette and provide the sql.

**Comment by JoyBaule** [comment]
@andreabringhurst Here is my narrative to update the FTE vignette for the request to create IPEDS 12 Month Enrollment FTE and USHE Annualized FTE:

### Introduction

Annualized FTE and 12 Month Enrollment FTE are metrics used to standardize the measurement of student enrollment in an academic year.

### USHE Annualized FTE

USHE Annualized FTE is determined by summing the total of each undergraduate and graduate FTE from the three end-of-term snapshots for each academic term within a reporting year (e.g., Summer 2023, Fall 2023, Spring 2024 for the USHE Reporting Year 2023-24) and then dividing this total by 2. This calculation has been verified by Matt Baxter and Beckie Hermanson at the USHE System Office in November 2024.


```sql
#USHE Annualized FTE SQL placeholder

```

```{r, echo=FALSE}
ushe_annualized_fte <- utHelpR::get_data_from_sql_file("ushe_annualized_fte.sql", dsn="edify", context="project")

```

```{r, echo=FALSE}
ushe_fte_test <- ushe_annualized_fte %>% 
  filter(ushe_academic_year_code != '2025') %>% 
  group_by(ushe_academic_year_code) %>% 
  summarise(ushe_annualized_eot_fte = round(sum(eot_total_fte) / 2, 0))
```

### IPEDS 12 Month Enrollment

IPEDS 12 Month Enrollment FTE is calculated by summing the number of attempted credits for undergraduate and graduate students from the three census snapshots for each academic term within a reporting year (e.g., Summer 2023, Fall 2023, Spring 2024 for the IPEDS data collection period 2023-24, reported in the 2024-25 IPEDS survey series). The undergraduate total credits attempted are divided by 30 (fifteen times two), and the graduate credits attempted are divided by 24 (twelve times two) to calculate the 12 Month Enrollment FTE by academic year. This calculation is verified in the 2024-25 data collection tutorial of the 12-Month Enrollment Component/FTE Enrollment-Credit Hours training materials provided by the Association for Institituional Research(AIR).

```sql
#IPEDS Twelve Month FTE SQL placeholder

```

```{r, echo=FALSE}
ipeds_twelve_month_fte <- utHelpR::get_data_from_sql_file("ipeds_twelve_month_fte.sql", dsn="edify", context="project")
```

### Summary

Due to differences in computation methods and the versions used for each calculation, the USHE Annualized FTE and IPEDS 12-Month Enrollment FTE will not perfectly align in historical comparisons. Additionally, changes in reporting guidance for USHE and IPEDS over time make it challenging to recreate each previously reported calculation. However, a reasonable comparison can still be made using current guidance computations.

The USHE Annualized FTE is based on End of Term data for each academic term included in the academic year, while the IPEDS 12-Month Enrollment FTE is based on Census data for each academic term within the same period.

Although these two FTE calculations are not expected to match exactly, USHE unofficially expects to see at least a 90% alignment between the two reported FTE numbers.

2023-24 Comparison: 

- 2023-24 USHE Annualized EOT FTE is 9054.
- 2023-24 IPEDS 12 Month Census FTE is 9046.

**Comment by andreabringhurst** [comment]
I'm going to close this. This issue is being addressed in pull #52
